src/epochs.lua

```diff
@@ -370,26 +364,19 @@ function epochs.createEpoch(timestamp, blockHeight, hashchain)
 	assert(type(hashchain) == "string", "Hashchain must be a string")
 
 	local epochIndex = epochs.getEpochIndexForTimestamp(timestamp)
-	if next(epochs.getEpoch(epochIndex)) then
+	if epochs.getEpoch(epochIndex) then
 		-- silently return
-		print("Epoch already exists for index: " .. epochIndex)
-		return
+		print("Epoch already exists for index. Returning existing epoch: " .. epochIndex)
+		return nil
```

Comment:
The comment above says "returning existing" but we're returning nil. Is that intended?

Reply:
modified here with a note why not to return the epoch - [c8472ba](https://github.com/ar-io/ar-io-network-process/pull/338/commits/c8472ba416686d262eda4e32d5819957931464d9)